src/app.module.ts

```diff
@@ -1,9 +1,22 @@
 import { Module } from '@nestjs/common';
-import { AppController } from './app.controller';
-import { AppService } from './app.service';
+import { ConfigModule } from '@nestjs/config';
+import { SequelizeModule } from '@nestjs/sequelize';
 
 @Module({
-  controllers: [AppController],
-  providers: [AppService],
+  controllers: [],
+  providers: [],
+  imports: [
+    ConfigModule.forRoot(),
+    SequelizeModule.forRoot({
+      dialect: 'postgres',
+      host: 'localhost',
```

Comment:
Please move it to .env also. Same for port and db name. 
autoloadmodels can break app for production, remove it. All db updates should be done via migrations.